docs/source/Cookbooks/01_comingsoon.md

```diff
@@ -0,0 +1,6 @@
+# Coming soon
```

Comment:
nit: `Cookbooks` we could go with lower cases only for folder names

Reply:
more as a follow-up, all the directories have a capital letter. Doesn't really change anything, but if we go with all small-caps, we need to update the script creating new roles.